src/app/modules/payments/__tests__/stripe.controller.spec.ts

```diff
@@ -59,6 +59,7 @@ describe('stripe.controller', () => {
         ...mockBusinessInfo,
         formTitle: mockFormTitle,
         submissionId: mockSubmissionId,
+        background: false,
```

Comment:
just noting: interesting choice of putting `false` here when default is true 😂

Reply:
I kinda rushed this out, but will look at it properly (if no choice)

Reply:
D: but yea I guess, just wait for JTC to get back ig